parts/configure-swarm-cluster.sh

```diff
@@ -6,6 +6,7 @@ echo "starting swarm cluster configuration"
 date
 ps ax
 
+# **NOTE** Keep version numbers in sync with constants in cmd/info.go
 SWARM_VERSION="swarm:1.1.0"

```

Comment:
"The sky's the limit" :)
I tried to keep it simple, but your guys perfectionism took over :)